src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateTargetCoverage.java

```diff
@@ -34,12 +35,42 @@
  * Calculate read-counts across targets given their reference coordinates.
  *
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ *
+ * <h3>Examples</h3>
+ *
+ * <p>
+ *     The command encompasses empirically determined parameters for TCGA project data.
+ *     You may obtain better results with different parameters.
+ * </p>
+ *
+ * <p>For whole exome sequencing (WES) data: </p>
+ *
+ * <pre>
+ * java -Xmx4g -jar $gatk_jar CalculateTargetCoverage \
+ *   --input sample.bam \
+ *   --targets padded_targets.tsv \
```

Comment:
It might help to explicitly write down the header and a line or two of the input targets file format.  I think some of our other CNV tools do this in their javadoc.